Delete :intellij Gradle subproject (#4412)
aws/aws-toolkit-jetbrains: CONTRIBUTING.md

### Instructions

1. Clone the github repository.

> 1. To manually build a plugin distribution, run the `buildPlugin` task on the relevant Gradle project. 

Reviewer: ```suggestion
2. To manually build a plugin distribution, run the `buildPlugin` task on the relevant Gradle project. 
```

   For example, `./gradlew :plugin-toolkit:intellij-standalone:buildPlugin` will produce a plugin zip under `plugins/toolkit/intellij-standalone/build/distributions`.

>    You can also run the `:plugin-core:buildPlugin` and `:plugin-amazonq:buildPlugin` tasks

Reviewer: ```suggestion
   You can also run the `:plugin-core:buildPlugin` and `:plugin-amazonq:buildPlugin` tasks
   Use the `-PideProfileName={JETBRAINS_VERSION}` option to build the plugin for a particular IDE version (e.g `./gradlew :plugin-toolkit:intellij-standalone:buildPlugin -PideProfileName=2024.1`)
```

> 1. In your JetBrains IDE (e.g. IntelliJ) navigate to the `Plugins` preferences and select "Install Plugin from Disk...", navigate to the zip file(s) produced in step 1. 

Reviewer: ```suggestion
3. In your JetBrains IDE (e.g. IntelliJ) navigate to the `Plugins` preferences and select "Install Plugin from Disk...", navigate to the zip file(s) produced in step 2. 
```

> 1. You will be prompted to restart your IDE.

Reviewer: ```suggestion
4. You will be prompted to restart your IDE.
```
